### DISCUSSION TITLE
Multi monitors

### DISCUSSION THREAD

**Discussion by eojonathan** [Taiizor/Sucrose#88] [Q&A] [answered]
Primeiramente, obrigado pelo projeto.

Sou iniciante no mundo do Git mas venho deixar uma dúvida, na descrição diz ser compatível com vários monitores mas não achei essa configuração, uso 3 monitores e ele só altera o wallpaper do monitor principal.

Alguém sabe como fazer?
Ainda assim, obrigado pela ajuda.

--

Firstly, thank you for the project.

I'm new to the world of Git but I have a question, in the description it says it's compatible with multiple monitors but I didn't find this configuration, I use 3 monitors and it only changes the wallpaper on the main monitor.

Does anyone know how to do it?
Still, thanks for the help.

**Comment by Taiizor** [comment, marked as answer]
Um problema já existente foi aberto sobre este assunto [aqui](https://github.com/Taiizor/Sucrose/issues/81).

Atualmente, você só pode definir papel de parede para uma tela selecionada ou para cobrir todas as telas.
![Sucrose](https://github.com/Taiizor/Sucrose/assets/41683699/0711c5c9-7167-49b7-b87d-26afcc70e17f)
![Sucrose](https://github.com/Taiizor/Sucrose/assets/41683699/95c4f714-ed71-452e-8b51-c10e73804926)

A opção de duplicar o mesmo papel de parede em todas as telas está desativada, como pode ser visto nas imagens acima, pois estou trabalhando nisso.

---

A pre-existing [issue](https://github.com/Taiizor/Sucrose/issues/81) has already been opened regarding this matter.

Currently, you can only set a wallpaper for a selected screen only or to expand across screens.
![Sucrose](https://github.com/Taiizor/Sucrose/assets/41683699/2f171c18-f494-4ee0-8f48-0064efc38157)
![Sucrose](https://github.com/Taiizor/Sucrose/assets/41683699/75fc1aaf-8f37-4c89-9ca0-25df0047e495)

The option to duplicate same wallpaper is disabled, as seen in the images above, because I am working on it.
